server/js/sockets.js

```diff
@@ -154,6 +155,24 @@ sockets.init = function socketInit(server) {
             `Error in parsing received payload\n\ttopic: ${topic}\n\tpayload: ${payloadString}\n`,
           );
         }
+      } else if (topic.startsWith(WirelessModule.base)) {
+        // Emit on appropriate channel
+        try {
+          const topicString = topic.split('/').slice(1); // Remove leading ""
+          // topicString: ["v3", "wireless_module", <id>, <property>]
+          const value = JSON.parse(payloadString);
+          const id = topicString[2];
+          const property = topicString[3];
```

Comment:
`const [, , , id, property] = topic.split('/');` would be more expressive here.

Reply:
I completely forgot about array deconstruction assignment. Good pick up!
736b2c5